### ISSUE TITLE
Wanted Poster (185035) (BB #931)

### ISSUE THREAD

**Issue by OregonAsari** [OregonCore/OregonCore#931]
This issue was migrated from bitbucket.
**Original Reporter:** bewtihozzy
**Original Date:** 29.05.2015 20:34:12 GMT+0000
**Original Priority:** major
**Original Type:** bug
**Original State:** resolved
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/931

<hr>

Unable to open the quest from Wanted Poster (185035) in Sylvannar (Blade edge mountains)

Sorry if the description is to short.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** bewtihozzy
**Original Date:** 29.05.2015 22:13:02 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/931/#comment-18472605

<hr>

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 30.05.2015 01:22:31 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/931/#comment-18473899

<hr>

[Core/Opcodes] Fix a sanity check in HandleGameObjectUseOpcode that was added to
               block certain exploits. As far as I'm aware, the fix is still valid
               but now we have returned a majority of broken gameobjects to normal.

```
           Fixes #931, #943.
           Correctly fixes #912, #909 and #898.
```

→ <<cset e215d2efab6f>>
